MANIFEST.in

```diff
@@ -1,9 +1,12 @@
 include *.py
 include *.txt
+include *.rst
+include .coveragerc
+include tox.ini
```

Comment:
It depends on whether you think sdists are meant for machines to build, or for people to use.  While most people will use git clones, sometimes you need to use an sdist (e.g. you need to backport a quick fix onto a pinned version, and it's simpler to grab the sdist from PyPI than to hunt down the exact branch/tag/commit from whatever repo upstream uses).

In those circumstances it's helpful to provide files like .coveragerc or tox.ini (or even .travis.yml) so people can look at them and see what's the procedure to run tests for this particular piece of software.